spec/granite_spec.cr

```diff
@@ -8,6 +9,30 @@ describe Granite::Base do
     model.is_alive.should be_true
   end
 
+  describe Logger do
+    describe "when logger is set to IO" do
+      it "should be logged as DEBUG" do
+        IO.pipe do |r, w|
+          Granite.settings.logger = Logger.new(w, Logger::Severity::DEBUG)
+
+          Person.first
+
+          r.gets.should match /D, \[.*\] DEBUG -- : .*SELECT.*people.*id.*FROM.*people.*LIMIT.*1.*: .*\[\]/
+        end
+      end
+    end
+
+    describe "when logger is set to nil" do
+      it "should not be logged" do
+        Granite.settings.logger = Logger.new nil
+
+        a = 0
+        Granite.settings.logger.info { a = 1 }
+        a.should eq 0
+      end
```

Comment:
this test is essentially testing the functionality of Logger, I'm not sure it's needed.

Reply:
Probably not, but I'd rather keep it.  Completes the idea of `if you have a logger defined it logs, otherwise it doesn't`.